backend/btrixcloud/colls.py

```diff
@@ -386,7 +388,11 @@ async def add_collection(
         new_coll: CollIn, org: Organization = Depends(org_crawl_dep)
     ):
         return await colls.add_collection(
-            org.id, new_coll.name, new_coll.crawlIds, new_coll.description
+            org.id,
+            new_coll.name,
+            new_coll.crawlIds,
+            new_coll.description,
+            new_coll.isPublic,
```

Comment:
Might be better to pass in the whole new_coll `CollIn` object here rather than including all the separate fields